init.js

```diff
@@ -2,22 +2,13 @@
 
 const path = require('path')
 const Module = require('module')
-const telemetry = require('./packages/dd-trace/src/telemetry/init-telemetry')
 const semver = require('semver')
+const { isTrue } = require('./packages/dd-trace/src/util')
+const telemetry = require('./packages/dd-trace/src/telemetry/init-telemetry')
 
-function isTrue (envVar) {
-  return ['1', 'true', 'True'].includes(envVar)
-}
-
-// eslint-disable-next-line no-console
-let log = { info: isTrue(process.env.DD_TRACE_DEBUG) ? console.log : () => {} }
-if (semver.satisfies(process.versions.node, '>=16')) {
-  const Config = require('./packages/dd-trace/src/config')
-  log = require('./packages/dd-trace/src/log')
-
-  // eslint-disable-next-line no-new
-  new Config() // we need this to initialize the logger
-}
+const log = semver.satisfies(process.versions.node, '>=16')
+  ? require('./packages/dd-trace/src/log')
+  : { info: isTrue(process.env.DD_TRACE_DEBUG) ? console.log : () => {} } // eslint-disable-line no-console
```

Comment:
Is the Node.js version check still required now that we no longer support Node.js <16? If not, we could just always require `./packages/dd-trace/src/log`.

Reply:
sounds reasonable, i've also removed a >=16 check somewhere else in the code so i'd say we can remove this one too ? unless someone knows about an edge case idk

Reply:
I've removed in b376911 👍 

@bengl are you ok with this as well?

Reply:
Fine with removing >=v16 check 👍